tests/handlebars/html-whitespace-sensitivity/jsfmt.spec.js

```diff
@@ -1,8 +1,83 @@
+const { outdent } = require("outdent");
+
 run_spec(__dirname, ["glimmer"], {
   htmlWhitespaceSensitivity: "ignore",
   printWidth: 40,
 });
-run_spec(__dirname, ["glimmer"], {
-  htmlWhitespaceSensitivity: "strict",
-  printWidth: 40,
-});
+run_spec(
+  {
+    dirname: __dirname,
+    snippets: [
+      // single
+      ...[
+        // https://developer.mozilla.org/en-US/docs/Glossary/Whitespace#In_HTML
+        "\u0009",
+        "\u000A",
+        "\u000C",
+        "\u000D",
+        "\u0020",
+      ].map((textContent) => ({
+        code: `<div>${textContent}</div>`,
+        name: "content",
+        output: "<div>\n</div>",
+      })),
+
+      // many
+      {
+        code: "<div>\u0009\u000A\u000C\u000D\u0020</div>",
+        output: "<div>\n\n</div>",
```

Comment:
Shouldn't this be `<div>\n</div>` ?

Reply:
`\u0009\u000A\u000C\u000D\u0020` includes 2 newlines, which are preserved. From a browser perspective, `<div>\n</div>` and `<div>\n\n</div>` will be rendered the same way. But from an author perspective, it can be nice sometimes to keep the additional newline for readablility purpose. For instance:

```hbs
<div>
    <div>
    </div>

    <div>
    </div>
</div>
```
wdyt?

Reply:
please note that in this implementation, 3 (or more) newlines will be reduced to two

Reply:
That's inconsistent with how Prettier treats empty lines everywhere else: https://prettier.io/docs/en/rationale.html#empty-lines

Reply:
@thorn0 I was not aware of the the [empty lines doc](https://prettier.io/docs/en/rationale.html#empty-lines), I have been somehow lucky :)